react-components/src/hooks/network/getPointCloudModelsForAsset.test.ts

```diff
@@ -0,0 +1,145 @@
+import { beforeEach, describe, expect, test, vi } from 'vitest';
+import { getPointCloudModelsForAsset } from './getPointCloudModelsForAsset';
+import { Mock } from 'moq.ts';
+import {
+  type AnnotationsAssetRef,
+  type CursorAndAsyncIterator,
+  type Revision3D,
+  type CogniteClient
+} from '@cognite/sdk';
+import { type DmsUniqueIdentifier } from '../../data-providers';
+import { createCursorAndAsyncIteratorMock } from '#test-utils/fixtures/cursorAndIterator';
+
+describe(getPointCloudModelsForAsset.name, () => {
+  const ASSET_ID = 123;
+  const DMS_ID: DmsUniqueIdentifier = { externalId: 'some-external-id', space: 'some-space' };
+  const ARBITRARY_MODEL_ID = 12345678;
+  const ARBITRARY_REVISION_ID = 87654321;
+
+  const ARBITRARY_REVISION: Revision3D = {
+    id: ARBITRARY_REVISION_ID,
+    createdTime: new Date(),
```

Comment:
non blocking
would be preferred to initialize it with a date as `new Date()` is non deterministic